docker-compose.review.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.4'
+services:
+  redis:
+    image: redis:3.2-alpine
+    restart: unless-stopped
+  postgres:
+    image: mdillon/postgis:9.6-alpine
+    environment:
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD:-password}"
+      POSTGRES_USER: "${POSTGRES_USER:-postgres}"
+    volumes:
+      - db-data:/var/lib/postgresql/data:cached
+    ports:
+      - "54320:5432"
+    restart: unless-stopped
+  vets-api:
+    build:
+      context: .
+      target: development
+      args:
+        exclude_sidekiq_ent: "${EXCLUDE_SIDEKIQ_ENTERPRISE:-false}"
+        sidekiq_license: "${BUNDLE_ENTERPRISE__CONTRIBSYS__COM}"
+        userid: "${VETS_API_USER_ID}"
+    command: >
+      bash -c "bundle exec rake db:migrate || bundle exec rake db:setup db:migrate
```

Comment:
Is the idea here to only run a setup on first deploy and migrations on PR commits thereafter?